extensions/amazon-lambda/maven-archetype/src/main/resources/archetype-resources/build.gradle

```diff
@@ -30,3 +30,26 @@ java {
     targetCompatibility = JavaVersion.VERSION_1_8
 }
 
+buildNative {
+    enableHttpUrlHandler = true
+}
```

Comment:
Nice one.  Maven seemed to have this enabled by default (i.e. you'd only be adding this for HTTPS).  It was truly missing for the Gradle native-image?  

It's a shame Gradle hides the output when running native-image, unlike Maven, where you get the feedback.

Reply:
It was outputted by quarkus at Runtime but not by Gradle during build.